standard/sections/clause_7_monitoring_event_topic.adoc

```diff
@@ -46,14 +43,10 @@ The table below provides an overview of the primary topic levels.
 |4
 |centre-id
 |Acronym proposed by Member and endorsed by WMO Secretariat, of the entity producing the event (source)
-
-|5
-|centre-id
-|Acronym proposed by Member and endorsed by WMO Secretariat, of the subject of the event (subject)
 |===
 
 ==== Publishing
 
-A simple ruleset is defined for publishing events to WMET that enables the clear identification of the event producer and its subject, to trigger corrective action.
+A simple ruleset is defined for publishing events to WMET that enables the clear identification of the event producer, to trigger corrective action.
```

Comment:
A simple ruleset is defined for publishing events to WMET that enables the clear identification of the event subject, to trigger corrective action or as an informative notice.